samples/multicluster-razee/README.md

```diff
@@ -0,0 +1,481 @@
+* The `selector/workloadSelector` field matches the labels of our Jupyter notebook workload.
+* The `data` field includes a `dataSetID` that matches the asset identifier in the catalog.
+* The `protocol` indicates that the developer wants to consume the data using Apache Arrow Flight. For some protocols a `dataformat` can be specified as well (e.g., `s3` protocol and `parquet` format).
+
+Run the following command to wait until the `FybrikApplication` is ready:
+
+```bash
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.conditions[?(@.type == "Ready")].status}') != "True" ]]; do echo "waiting for fybrik-notebook-sample/paysim-csv asset" && sleep 5; done
+```
+
+## Read the dataset from the notebook
+
+In your **terminal**, run the following command to print the endpoint to use for reading the data. It fetches the code from the `FybrikApplication` resource:
+
+```bash
+kubectl config use-context kind-control
+ENDPOINT_SCHEME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.scheme})
+ENDPOINT_HOSTNAME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.hostname})
+ENDPOINT_PORT=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.port})
+printf "${ENDPOINT_SCHEME}://${ENDPOINT_HOSTNAME}:${ENDPOINT_PORT}"
+```
+
+The next steps use the endpoint to read the data in a python notebook
+
+1. Insert a new notebook cell to install pandas and pyarrow packages:
+  ```python
+  %pip install pandas pyarrow
+  ```
+2. Insert a new notebook cell to read the data using the endpoint value extracted from the `FybrikApplication` in the previous step:
+  ```bash
+  import json
+  import pyarrow.flight as fl
+  import pandas as pd
+
+  # Create a Flight client
+  client = fl.connect('<ENDPOINT>')
+
+  # Prepare the request
+  request = {
+      "asset": "fybrik-notebook-sample/paysim-csv",
+      # To request specific columns add to the request a "columns" key with a list of column names
+      # "columns": [...]
+  }
+
+  # Send request and fetch result as a pandas DataFrame
+  info = client.get_flight_info(fl.FlightDescriptor.for_command(json.dumps(request)))
+  reader: fl.FlightStreamReader = client.do_get(info.endpoints[0].ticket)
+  df: pd.DataFrame = reader.read_pandas()
+  ```
+4. Insert a new notebook cell with the following command to visualize the result:
+  ```
+  df
+  ```
+5. Execute all notebook cells and notice that the `nameOrig`, `oldbalanceOrg`and `newbalanceOrig` columns appear redacted.
+
```

Comment:
What about add the clean up step and mention that all these components will be clean up if we delete fybrikapplication. I saw the notebook example has the cleanup step.

Reply:
I've considered this, but I don't think it's a high priority because the [notebook sample](https://fybrik.io/dev/samples/notebook/) is done with any kubernetes cluster, e.g. kind or a real kubernetes cluster. For a real cluster, cleanup is necessary. 
But this sample setups a kind multi-cluster, these clusters are not really useful, can be deleted after the experiment. So I think "clean up" is not necessary.
Anyway, it‘s nice to have. I think I can add it.

Reply:
Done. Please take another look.